tic_tac_toe/game.py

```diff
@@ -87,18 +128,54 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
-
-
+    if not _position_is_valid(position):
+        raise InvalidMovement("Position out of range.")
+
+    if _board_is_full(game['board']) or game['winner'] is not None:
+        raise InvalidMovement("Game is over.")
+        
+    if game['next_turn'] != player:
+        raise InvalidMovement('"{}" moves next'.format(game['next_turn']))
+    
+    if not _position_is_empty_in_board(position, game['board']):
+        raise InvalidMovement("Position already taken.")
+    
+    
+        
+    # Make a move
+    game['board'][position[0]][position[1]] = player
+    if get_next_turn(game) == 'X':
+        game['next_turn'] ='O'
+    else:
+        game['next_turn'] = 'X'
+    
+    
+    result = _check_winning_combinations(game['board'], player)
+    if result is not None:
```

Comment:
Since `_check_winning_combinations()` returns `None` when there is not yet a winner, it would be sufficient here to check `if result:`.